templates/4b_normalize_image_data.ipynb

```diff
@@ -131,9 +131,18 @@
     "for fov in fovs:\n",
```

Comment:
<p>I think its better to leave this out for now. We shouldn't do any processing of the proficient tifs. They're only there in case something changes in the future. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/toffy/pull/342/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
<p>We'll also need to remove this call from <code>watcher_callbacks.generate_pulse_heights</code> as well, since it also calls <code>write_mph_per_mass</code> a second time for mass proficient MPH normalization.</p>